test/Algebra/Graph/Test/Acyclic/AdjacencyMap.hs

```diff
@@ -0,0 +1,235 @@
+-------------------------------------------------------------
+-- |
+-- Module     : Algebra.Graph.Test.Acyclic.AdjacencyMap
+-- Stability  : experimental
+--
+-- Testsuite for "Algebra.Graph.Acyclic.AdjacencyMap".
+-------------------------------------------------------------
+
+module Algebra.Graph.Test.Acyclic.AdjacencyMap (
+  testAcyclicAdjacencyMap 
+  ) where
+
+import Algebra.Graph.Acyclic.AdjacencyMap
+import Algebra.Graph.Test
+import Algebra.Graph ()
```

Comment:
Why do we need this import?

Reply:
I think I am (was) using the `Num` instance of `Graph` somewhere. I'll check and remove it if I can.